lblod/frontend-gelinkt-notuleren
# Drop requirement to pass template to its own `loadBody` method

### Overview
This small PR does two things:
- rename `loadTemplateBody` method on regulatory statement templates to `loadBody` for clarity
- remove the requirement to pass template object to itself in the `loadBody` method, as it is unnecessary and quite confusing.

### How to test/reproduce
- Open up the app
- Create a new regulatory statement
- Ensure the template is correctly loaded
